bot/extensions/polls/events.py

```diff
@@ -0,0 +1,43 @@
+import discord
+from discord.ext import commands
+
+from bot import core
+
+
+class PollEvents(commands.Cog):
+    """Events for polls in discord."""
+
+    def __init__(self, bot: core.DiscordBot):
+        self.bot = bot
+        self.emojis = ["1️⃣", "2️⃣", "3️⃣", "4️⃣", "5️⃣", "6️⃣", "7️⃣", "9️⃣", "🔟"]
+
+    def poll_check(self, message: discord.Message):
+        try:
+            embed = message.embeds[0]
+        except Exception:
+            return False
+        if str(embed.footer.text).count("Poll by") == 1:
+            return message.author == self.bot.user
+        return False
```

Comment:
Avoid redefining this function in 3 places, and also improve this except statement.

Reply:
its only been defined twice so i think its fine, except statement has been completely removed in the newest commit